ekglib/dataset/various.py

```diff
@@ -110,10 +110,13 @@ def export_graph(
     chunk = bytearray(chunk_max_size)
     chunk_size = chunk_max_size
     buf.seek(0)
+
     while chunk_size == chunk_max_size:
         chunk_size = buf.readinto(chunk)
+        log_item(f"{s3_file_name} chunk_size", str(chunk_size))
```

Comment:
ah good. that's the extra debug I was talking about. though hopefully your fix alone was the problem 🤞